src/pages/Home/index.js

```diff
@@ -127,8 +127,8 @@ const Cards = ({ size }) => (
     <Card
       logo="/img/munch-and-learn-3.jpg"
       title="Introducing HPE DEV Munch & Learn series"
-      desc="Session 3: Data Science Unplugged Part 1"
-      link=" https://hpe.zoom.us/meeting/register/tJYudO2uqD8iGddvDqH__n4v3lgzyAg42FrL"
+      desc="Session 4: Building a foundation for zero trust with SPIFFE"
```

Comment:
Just add a period in the end. Session 4: Building a foundation for zero trust with SPIFFE.

Reply:
Thanks! Updated here: https://github.com/HewlettPackard/HackShack-Session-Landing-Page/pull/159/commits/512153d216de7c1209f2e4d8880c6278068b3ebe